cmd/bb_storage/main.go

```diff
@@ -40,6 +52,32 @@ func main() {
 		log.Fatal(http.ListenAndServe(*webListenAddress, nil))
 	}()
 
+	pe, err := prometheus_exporter.NewExporter(prometheus_exporter.Options{
```

Comment:
We likely want to add functionality like this to other bb-* processes as well, right? If so, could we please add this to some `pkg/*`?